src/Illuminate/Foundation/Testing/Concerns/InteractsWithContainer.php

```diff
@@ -109,8 +109,17 @@ protected function withoutVite()
             $this->originalVite = app(Vite::class);
         }
 
-        $this->swap(Vite::class, function () {
-            return '';
+        $this->swap(Vite::class, new class
+        {
+            public function __invoke()
+            {
+                return '';
+            }
+
+            public function reactRefresh()
+            {
+                return '';
+            }
```

Comment:
What do you think of this?

```suggestion
            public function __call($name, $arguments)
            {
                return '';
            }
```

This will help if any new methods are added, as I can see people forgetting to add them here.

cc @timacdonald